documentation/Pipfile

```
@@ -12,6 +12,7 @@ sphinx-notfound-page = "~=1.0"
 furo = ">=2023.8.19"
 myst-parser = "~=2.0"
```

This is a clearer way to establish where `linkify-it-py` is being used imo.

```suggestion
myst-parser = {version = "~=2.0", extras = ["linkify"]}
```

This change will necessitate running `pipenv lock` again.

---

Hi @dhruvkb thanks for your insight on this, would declaring `linkify` as a myst-parser dependency as suggested require me to remove the `linkify-it-py = "*"` dependency declaration from the pipfile entirely or it can remain alongside the new change?

---

You can remove the explicit declaration from line 15. By using the `extras`, pipenv will automatically add `linkify-it-py` as a sub-dependency of `myst-parser`.

---

Thank you, the PR has been updated.